### PR TITLE
updating-llvm: keep a calm tone

### PR DESCRIPTION
**Pull request by tshepang**
Also, keep it trimmed... if a formal policy is adopted, this text can be updated to indicate so.

### CODE REVIEW

File: src/backend/updating-llvm.md [hunk 1]
```diff
@@ -2,20 +2,15 @@
...
-
-* We try to always support the latest released version of LLVM
-* We try to support the "last few" versions of LLVM (how many is changing over
-  time)
-* We allow moving to arbitrary commits during development.
-* Strongly prefer to upstream all patches to LLVM before including them in
-  rustc.
-
-This policy may change over time (or may actually start to exist as a formal
-policy!), but for now these are rough guidelines!
```
Comment (JohnTitor) [inline src/backend/updating-llvm.md:18]: Why did you remove this sentence? This caution looks useful to me.

File: src/backend/updating-llvm.md [hunk 2]
```diff
@@ -2,20 +2,15 @@
...
+<!-- date-check: Aug 2022 -->
+There is no formal policy about when to update LLVM or what it can be updated to,
+but a few guidelines are applied:
+
+* We try to always support the latest released version
+* We try to support the last few versions of LLVM
+  (and the number changes over time)
```
Comment (JohnTitor) [inline src/backend/updating-llvm.md:11]: The former removed "of LLVM" whereas the latter keeps it, it's inconsistent.

File: src/backend/updating-llvm.md [hunk 3]
```diff
@@ -2,20 +2,15 @@
...
+* We allow moving to arbitrary commits during development
+* We Strongly prefer to upstream all patches to LLVM before including them in rustc
```
Comment (JohnTitor) [inline src/backend/updating-llvm.md:13]:
```suggestion
* We strongly prefer to upstream all patches to LLVM before including them in rustc
```

**Review by JohnTitor — COMMENTED** [review]
Hmm, I'm not sure how this improves stuff. It doesn't seem that the current is losing readability. 
If we prefer a "calm tone", we should make a policy to do that. Otherwise, this change would be easily dropped/reverted someday (note that many contributors would edit this guide in the future).

**Comment by JohnTitor** [comment]
Triage: @tshepang Could you check the review comments when you have time?

File: src/backend/updating-llvm.md (hunk 1 above)
Comment (tshepang) [reply src/backend/updating-llvm.md:18]: It's basically repeated, so I keep it in one place... see line 6 of new changes.

File: src/backend/updating-llvm.md [hunk 4]
```diff
@@ -2,20 +2,15 @@
...
-  rustc.
-
-This policy may change over time (or may actually start to exist as a formal
-policy!), but for now these are rough guidelines!
+<!-- date-check: Aug 2022 -->
+There is no formal policy about when to update LLVM or what it can be updated to,
+but a few guidelines are applied:
+
+* We try to always support the latest released version
+* We try to support the last few versions of LLVM
```
Comment (tshepang) [inline src/backend/updating-llvm.md:10]:
```suggestion
* We try to support the last few versions
```

**Comment by tshepang** [comment]
I missed your review suggestions for some reason... now handled.

**Review by JohnTitor — APPROVED** [review]
I'm going to approve it as this is more of a "keep a calm tone" thing, i.e. includes minor clean-ups. Thanks!
